lib/worker.js

```diff
@@ -111,6 +112,25 @@ Worker.prototype._reportError = function (err) {
 }
 
 /**
+ * Helper function for creating monitor-dog events tags
+ * @private
+ * @param {Object} tags as Object {queue: 'docker.event.publish'}
```

Comment:
can you expand this jsdoc to illustrate what it's doing? I _think_ it's returning the queue name as a value on an object, but the reverse in there makes me curious...